src/main/java/com/microsoft/aad/msal4j/IntegratedWindowsAuthenticationParameters.java

```diff
@@ -21,24 +22,30 @@
 @Accessors(fluent = true)
 @Getter
 @AllArgsConstructor(access = AccessLevel.PRIVATE)
-public class IntegratedWindowsAuthenticationParameters implements IApiParameters {
+public class IntegratedWindowsAuthenticationParameters implements IAcquireTokenParameters {
```

Comment:
is extraHttpHeaders applicable for any implementation of IAcquireTokenParameters ?
if yes Abstract class should be introduced to avoid code duplication

Reply:
I took a look at this and since the parameter classes are Lombok builders,  IAcquireTokenParameters would have to be an abstract builder. If I remember correctly we couldn't get the Lombok annotations to play nicely with abstract builders in the ClientApplication builders, so they had to be written out. 

Updating this here would require the same change, and would actually result in writing a lot more code (writing out all of the Parameter builders) in exchange for avoiding some code duplication.